doc/whats_new/v0.23.rst

```diff
@@ -49,3 +49,16 @@ Changelog
 
 - |Fix| example fix in model XXX. :pr:`xxxx` or :issue:`xxxx` by
   :user:`name <user id>`
+
+
+:mod:`sklearn.ensemble`
+.......................
+
+- |Efficiency| add the option `class_weight="balanced_bootstrap"` in
+  :class:`ensemble.RandomForestClassifier` and
+  :class:`ensemble.ExtraTreesClassifier`. This option will ensure that the
+  number of samples in each bootstrap sample given to each tree of the ensemble
+  will be identical. This algorithm is also known as balanced random-forest
+  when applied to random-forest.
+  :pr:`13227` by :user:`Eric Potash <potash>`, :user:`Christos Aridas <chkoar>`
+  and :user:`Guillaume Lemaitre <glemaitre>`.
```

Comment:
same here, we should distinguish between a training sample, and a subsample (subset from the training set).

Also the description here only says that the subsets are of the same size. Shouldn't it say that the class proportions are respected (i.e. it's using stratification?)